proto/api/v2/compliance_integration_service.proto

```diff
@@ -32,14 +32,14 @@ service ComplianceIntegrationService {
     // ListComplianceIntegrations lists all the compliance operator metadata for the secured clusters
     rpc ListComplianceIntegrations (RawQuery) returns (ListComplianceIntegrationsResponse) {
         option (google.api.http) = {
-            get: "/v2/complianceintegrations"
+            get: "/v2/compliance/integrations"
         };
     }
 
     // GetComplianceIntegration retrieves the compliance operator metadata for the specified cluster
     rpc GetComplianceIntegration (ComplianceIntegrationStatusRequest) returns (ComplianceIntegration) {
         option (google.api.http) = {
-            get: "/v2/complianceintegrations/{cluster_id}"
+            get: "/v2/compliance/integration/{cluster_id}"
```

Comment:
A couple of thoughts:
1. The resource URL component should be plural.
2. If we want to add GetcomplianceIntegration by its id, we will have to get creative.
3. Do we need this endpoint? Can we recommend consumers to use the `ListComplianceIntegrations` endpoint with cluster ID/cluster name query?